cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -451,6 +451,25 @@ def build_app_dependency_search(app_list):
             }
         ],
     },
+    {
+        "name": "Cloudflare",
+        "app_dependencies": [
+            {
+                "label": "Cloudflare App for Splunk",
+                "link": "https://splunkbase.splunk.com/app/4501"
+            }
+        ],
+        "macro_configurations": [
+            {
+                "macro_name": "cs_cloudflare",
+                "label": "Cloudflare Data",
+                "search_by": "sourcetype",
+                "search_values": "cloudflare:json",
+                "earliest_time": "-4h@h",
```

Comment:
Is 4 hours enough to make sure some data present in this duration?

Reply:
Yes